cmd/routed-eni-cni-plugin/cni.go

```diff
@@ -275,6 +278,34 @@ func add(args *skel.CmdArgs, cniTypes typeswrapper.CNITYPES, grpcClient grpcwrap
 	// dummy interface is appended to PrevResult for use during cleanup
 	result.Interfaces = append(result.Interfaces, dummyInterface)
 
+	if utils.IsStrictMode(r.NetworkPolicyMode) {
+		// Set up a connection to the ipamD server.
```

Comment:
`Set up a connection to the ipamD server` --> `Set up a connection to the network policy agent`